index.bs

```diff
@@ -907,7 +908,7 @@ The getter of the {{Sensor/timestamp!!attribute}} attribute returns
     The {{Sensor/stop()}} method must run these steps:
 
     1.  If |sensor_instance|.{{[[state]]}} is "idle", then return.
-    1.  Set |sensor_instance|.{{[[state]]}} to "idle".
+    1.  [=Queue a task=] to set |sensor_instance|.{{[[state]]}} to "idle".
```

Comment:
Similar to start(), looks like invocations would queue multiple state change tasks + invoke unregister sensor object multiple times, right?